### PR TITLE
Added support for kube-context flag, to specify kubernetes context

### PR DESCRIPTION
**Pull request by therahulbhati**
#1102 Added support of flag kube-context, to specify kubernetes context

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1595)

### CODE REVIEW

File: pkg/fission-cli/flag/flag.go [hunk 1]
```diff
@@ -72,6 +72,8 @@ var (
 
 	ClientOnly = Flag{Type: Bool, Name: flagkey.ClientOnly, Usage: "If set, the CLI won't connect to remote server"}
 
+	KubeContext = Flag{Type: String, Name: flagkey.KubeContext, Usage: "Kubernetes context in which command execution occurs", DefaultValue: ""}
```
Comment (vishal-biyani) [inline pkg/fission-cli/flag/flag.go:75]: Nitpick

```suggestion
	KubeContext = Flag{Type: String, Name: flagkey.KubeContext, Usage: "Kubernetes context to be used for the execution of Fission commands", DefaultValue: ""}
```

**Comment by vishal-biyani** [comment]
Hey @therahulbhati Except for one minor change - this looks good. please update that and I will merge this

**Comment by therahulbhati** [comment]
@vishal-biyani I have updated the usage string.

**Comment by vishal-biyani** [comment]
LGTM @therahulbhati - thanks for this PR.

**Comment by vishal-biyani** [comment]
CI passed in https://github.com/fission/fission/pull/1598 so merging and closing this one.

**Pull request merged**